### DISCUSSION TITLE
System block but allow ping.exe

### DISCUSSION THREAD

**Discussion by 0xmbe** [tnodir/fort#345] [Q&A]
Hi, I want to have general NT Kernel & System blocked but some functions enabled like ping.exe or tracert.exe. If NT Kernel & System is blocked then I can ping only internal network, on external I get General error because.
Where can I set this?
Blocked:
![image](https://github.com/user-attachments/assets/002a0ebd-008f-403d-acb2-85fa15636a7a)
Not blocked:
![image](https://github.com/user-attachments/assets/4628fce4-ab26-4aa3-9e5f-ebec5b57c34e)

**Comment by tnodir** [comment]
Hi and welcome.

Please look at https://github.com/tnodir/fort/issues/340

**Reply by 0xmbe** [reply]
Thank you for your work.
So I guess this is how Windows works so I can't allow only specific integrated system exe.
Could we make a filter that would allow only specific protocol id through?
If that won't go I will just rely on finding alternative apps.

**Reply by tnodir** [reply]
> Could we make a filter that would allow only specific protocol id through?

Not now. I'm working on it..

**Reply by tnodir** [reply]
> Could we make a filter that would allow only specific protocol id through?

You can use.Global Rule with v3.14.13
See also https://github.com/tnodir/fort/wiki/Rules
